WebAutopark.DataAccesLayer/Repositories/DetailRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using WebAutopark.DataAccesLayer.Entities;
+
+namespace WebAutopark.DataAccesLayer.Repositories
+{
+    class DetailRepository : BaseRepository<VehicleType>
+    {
+        private const string _queryGetAll = "SELECT * FROM Details";
```

Comment:
```suggestion
        private const string QueryGetAll = "SELECT * FROM Details";
```
Constant field is always written via Camel Case

Reply:
Yeah, my mistake. But... Camel Case or Pascal Case? I read that Pascal Case is recommended for constants.